apps/crawler/src/processors/events-processor.ts

```diff
@@ -0,0 +1,105 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { Connection, Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { EventHandlerContext } from '@subsquid/substrate-processor';
+import { Event } from '@entities/Event';
+import { EventName, EventPhase } from '@common/constants';
+import { ScanProcessor } from './scan-processor';
+import { SdkService } from '../sdk.service';
+import { UtilsService } from '@common/utils/utils.service';
+import { ProcessorConfigService } from '../processor.config.service';
+
+@Injectable()
+export class EventProcessor {
+  name = 'event';
+  private logger: Logger;
+  private processor: ScanProcessor;
+
+  constructor(
+    @InjectRepository(Event)
+    private modelRepository: Repository<Event>,
+    private utils: UtilsService,
+    protected connection: Connection,
+    protected sdkService: SdkService,
+    private processorConfigService: ProcessorConfigService,
+  ) {
+    this.processor = new ScanProcessor(
+      this.name,
+      this.connection,
+      processorConfigService.getDataSource(),
+      processorConfigService.getRange(),
+      processorConfigService.getTypesBundle(),
+    );
+
+    this.logger = new Logger('EventProcessor');
+
+    Object.values(EventName).forEach((name) => {
+      this.processor.addEventHandler(name, async (ctx) => {
+        await this.upsertHandler(ctx, name);
+      });
+    });
+
+    this.logger.log('Starting processor...');
+  }
+
+  private async upsertHandler(
+    ctx: EventHandlerContext,
+    eventName: string,
+  ): Promise<void> {
+    const { height, timestamp } = ctx.block;
+
+    const log = {
+      eventName,
+      blockNumber: height,
+      blockTimestamp: timestamp,
+    };
+
+    try {
+      const data = await this.getData(ctx);
+
+      if (data) {
+        await this.modelRepository.upsert(data, [
+          'block_number',
+          'event_index',
+        ]);
+      }
+
+      this.logger.verbose({ ...log });
+    } catch (err) {
+      this.logger.error({ ...log, error: err.message });
+    }
+  }
+
+  private getData(ctx: EventHandlerContext) {
+    const { block, event } = ctx;
+
+    const result = {
+      block_number: event.blockNumber,
+      event_index: event.indexInBlock,
+      section: event.section,
+      method: event.method,
+      phase: event.extrinsic
```

Comment:
I think should not create values for fields "just because before they were like this") Don't we have raw event phase value it response?